codeforamerica/vita-min
# Add Submission Type to State Information Service

## Link to pivotal/JIRA issue
[link](https://codeforamerica.atlassian.net/browse/FYST-1437)

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
The StateManifest XML attribute StateSubmissionTyp needs to differ from the return type for MD, so a new attribute on the State Information Service for the submission type has been created.

## How to test?
I don't think any testing is needed, but I wrote a test locally to check if things looked correct, but I don't think it should be checked in. I'm happy to check it in if people feel differently!
